sdk/resources/azure-resourcemanager-resource/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-client-sdk-parent</artifactId>
+        <version>1.7.0</version>
+        <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+    </parent>
+
+    <groupId>com.azure.resourcemanager</groupId>
+    <artifactId>azure-resourcemanager-resource-generated</artifactId>
+    <version>1.0.0-beta.1</version>
+    <packaging>jar</packaging>
+
+    <name>Microsoft Azure SDK for Resource</name>
+    <description>Provides operations for working with resources and resource groups.</description>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+    <licenses>
+        <license>
+            <name>The MIT License (MIT)</name>
+            <url>http://opensource.org/licenses/MIT</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <url>https://github.com/Azure/azure-sdk-for-java</url>
+        <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
+        <developerConnection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+    <developers>
+        <developer>
+            <id>microsoft</id>
+            <name>Microsoft</name>
+        </developer>
+    </developers>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core-management</artifactId>
```

Comment:
code generator will be improved to output `<!-- {x-version-update;com.azure:azure-core-management;dependency} -->` as well.